web/api/v1/api_test.go

```diff
@@ -443,6 +443,7 @@ func TestEndpoints(t *testing.T) {
 	ng := testEngine(t)
 
 	t.Run("local", func(t *testing.T) {
+		t.Parallel()
```

Comment:
I'm afraid this has no effect as the subtests run sequentially.
for such tests that take less than `10ms` I don't think we should be running them in parallel.

Same for `TestMIMEType_String`, `TestMIMEType_Satisfies` and others belo